src/toolchain/distributable.rs

```diff
@@ -115,6 +115,13 @@ impl<'a> DistributableToolchain<'a> {
         Ok(())
     }
 
+    pub(crate) fn components(&self) -> anyhow::Result<Vec<crate::dist::manifest::ComponentStatus>> {
```

Comment:
Style nit: let's import `ComponentStatus` instead of using a fully-qualified path for it?

Reply:
cf. #3643.